lib/transactions/todos/create.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Transactions
+  module Todos
+    class Create
+      include Dry::Transaction
+      include Dry::Monads[:result]
+      include Import[:todo_repository]
+
+      step :validate_params
+      step :create_todo
+
+      def validate_params(input)
+        contract = ::Contracts::Todos::Create.new
+        result = contract.call(input)
+        if result.success?
+          Success(result.to_h)
+        else
+          Failure(result.errors.to_h)
+        end
+      end
+
+      def create_todo(input)
+        todo = todo_repository.build(input)
+        if todo_repository.save(todo)
```

Comment:
I see we use `build` and `save` methods only here, I do not see any case where this is useful. Why don't we create a `create` method on the repository?